Account/FirefoxAccountConfiguration.swift

```diff
@@ -53,7 +56,7 @@ public protocol FirefoxAccountConfiguration {
 }
 
 public struct LatestDevFirefoxAccountConfiguration: FirefoxAccountConfiguration {
-    public init() {
+    public init(prefs: Prefs? = nil) {
```

Comment:
This change isn't needed if you remove the constructor from the `FirefoxAccountConfiguration`.